app/src/androidTest/resources/email.txt

```diff
@@ -0,0 +1 @@
+pytest-28f90ad8b4@restmail.net
```

Comment:
Note that this is not a permanent account, while running the tests via `pipenv run pytest`a new account will be created, written in this file and read after that for the test to be used to sync in Fenix app. 
Same for the password below.